src/functions/validateTime.js

```diff
@@ -34,7 +26,7 @@ export const validateTime = (bookings, date, start, end) => {
   const duration = moment.duration(endTime.diff(startTime));
   const hours = duration.asHours();
   const dateChosen = moment(date).startOf('date');
-  if (hours > 2) {
+  if (hours > 12) {
     alert('Maximum hours of booking per person is 2 hours.');
```

Comment:
is the maximum confirmed to be 12? if so, the error message should correspond to that too

Reply:
ohh no, i did that for development purpose but forgot to change that back to 2 😂